docker-compose.prod.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.9'
+
+services:
+  db:
+    networks:
+      - prod
+
+  s3:
+    networks:
+      - prod
+
+  tarpisto:
+    image: tarpisto/tarpisto:latest
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://localhost:9000/api/health']
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    depends_on:
+      db:
+        condition: service_healthy
+      s3:
+        condition: service_healthy
+    environment:
+      - PORT=9000
+      - PG_CONNECTION_STRING=postgresql://tarpisto:tarpisto@db:5432/tarpisto
```

Comment:
Although this is a well-known postgresql default port, I would still like to see it tied to something that can be changed, even if it's only for internal. Especially seeing as this will go to prod.

Reply:
Longer clarification for this is at `run-prod.sh` comment. This is meant only for local testing prod-like. Needs better naming: TODO.

Reply:
Renamed to have local in the name.